sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -145,6 +145,17 @@ class SQLQuerySuite extends QueryTest with BeforeAndAfterAll with SQLTestUtils {
       Seq(Row("1"), Row("2")))
   }
 
+  test("constant functions") {
```

Comment:
let's be consistent with @chenghao-intel's length pr, and move this test into DataFrameFunctionSuite.scala.